test/common/quic/envoy_quic_client_stream_test.cc

```diff
@@ -317,7 +317,7 @@ TEST_F(EnvoyQuicClientStreamTest, PostRequestAnd1xx) {
   size_t i = 0;
   // Receive several 10x headers, only the first 100 Continue header should be
   // delivered.
-  for (const std::string& status : {"100", "199", "100"}) {
+  for (const std::string status : {"100", "199", "100"}) {
```

Comment:
Why is this necessary?

Reply:
Reverted. Was pointed out to me that this was a gcc vs clang build thing.